src/react-queries/queries/thornode.ts

```diff
@@ -9,7 +9,8 @@ import { thorService } from '@shapeshiftoss/swapper/dist/swappers/ThorchainSwapp
 import { Ok } from '@sniptt/monads'
 import axios from 'axios'
 import { getConfig } from 'config'
-import type { ThorchainBlock, ThorchainMimir } from 'lib/utils/thorchain/lending/types'
+import type { ThorchainMimir } from 'lib/utils/thorchain/lending/types'
```

Comment:
nitpick: might as well move this guy if you feel like it, not really lending specific, though indeed only used there atm!

Reply:
Prefer to avoid the diff there if you don't mind